polyglot/piranha/src/lib.rs

```diff
@@ -290,7 +304,7 @@ impl FlagCleaner {
   }
 
   /// Performs cleanup related to stale flags
-  fn perform_cleanup(&mut self) {
+    fn perform_cleanup(&mut self) {
```

Comment:
Is this indentation correct? I would expect the function definition to be indented with the comment above and one level out from the body.